Demo/ChartIQApp/ViewControllers/Chart/IntervalsViewController.swift

```diff
@@ -239,27 +235,16 @@ extension IntervalsViewController: UITableViewDataSource {
   func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
     guard let tableSection = TableSection(rawValue: section) else { return 0 }
     return intervals[tableSection]?.count ?? 0
-    //return tableSection == .first ? 1 : intervals[tableSection]?.count ?? 0
   }
 
   func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
     guard let baseTableViewCell = tableView.dequeueReusableCell(withIdentifier: Const.BaseTableCell.cellId,
                                                                 for: indexPath) as? BaseTableCell,
-      let tableSection = TableSection(rawValue: indexPath.section) else { return UITableViewCell() }
-    if tableSection == .first {
```

Comment:
I was going to leave all the custom periodicity code in so it can easily be restored, but this is fine too. We'll just need to remember that there is code to look up!